config/initializers/new_framework_defaults_5_2.rb

```diff
@@ -0,0 +1,35 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I am honestly not sure if we need to do anything here, but I assume we have to flip them all in and then comment them out one by one if we run into problems.